### PR TITLE
Support .yml YAML files in ember-intl functionality

### PR DESCRIPTION
**Pull request by patocallaghan**
Follow up to https://github.com/lifeart/els-intl-addon/pull/5#issuecomment-1109375255

Issue: https://github.com/lifeart/ember-language-server/issues/383

While the "official" file extension for YAML is `.yaml` many resources online still use `.yml` as the file extension of YAML. The application I work in has been using `.yml` for years and we'd love to use this addon without having to rename the files.

Ember Intl itself supports both the `.yaml` and `.yml` extension.

### CODE REVIEW

File: test/bultin-addons/core/intl-providers-test.ts [hunk 1]
```diff
@@ -424,8 +434,50 @@ for (const asyncFsEnabled of testCaseAsyncFsOptions) {
       });
     });
 
-    describe('provide completion - YAML', () => {
-      it('should autocomplete root translation in handlebars', async () => {
+    describe('YAML - .yaml file extensions', () => {
```
Comment (patocallaghan) [inline test/bultin-addons/core/intl-providers-test.ts:437]: I've duplicated the `.yaml` tests into `.yml` tests too

**Comment by lifeart** [comment]
@patocallaghan, `3.0.29` Stable Ember Language Server published

**Comment by patocallaghan** [comment]
Thanks @lifeart 🎉 How soon does it take for this to be "published"? Should I reload my VSCode to pick up the new changes? (Sorry if these are silly questions)

**Comment by lifeart** [comment]
Should take about 5-15 minutes to be available in VSCode store, just search for it, and if it's installed - auto-update will be downloaded, restart is required

**Comment by patocallaghan** [comment]
Thanks!!

**Pull request merged**